staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -151,9 +151,9 @@ func dataHandler(ctx context.Context, dev *driver.CustomizedDev) {
 		go twinData.Run(ctx)
 
 		//handle status
-		getStates := &DeviceStates{Client: dev.CustomizedClient, DeviceName: dev.Instance.Name,
-			DeviceNamespace: dev.Instance.Namespace}
-		go getStates.Run(ctx)
+		//getStates := &DeviceStates{Client: dev.CustomizedClient, DeviceName: dev.Instance.Name,
+		//	DeviceNamespace: dev.Instance.Namespace}
+		//go getStates.Run(ctx)
```

Comment:
I would suggest that instead of commented code, we probably want to `TODO` description what needs to be done here, so that we can always see the previous code in git history. after all, i do not think commented code is useful here.